.github/workflows/go_tests.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           fetch-depth: 2
       - uses: actions/setup-go@v2
         with:
-          go-version: '1.16'
+          go-version: '1.18'
```

Comment:
Not really - we could stick it in a template, but I don't really think its worth it for cases like this, otherwise you end up with a proliferation of templates which can be tough to follow